docs/deployment/openshift/README.md

```diff
@@ -298,6 +298,17 @@ To deploy to a custom namespace (replacing `falcon-system` as desired):
 </details>
 </details>
 
+## Upgrading
```

Comment:
@evanstoner here's the upgrade verbiage for operator capability level 2. I haven't done anything with OLMv1 yet... so not sure if we should do that here as well?